lib/cli/run.js

```diff
@@ -0,0 +1,403 @@
+'use strict';
+
+/**
+ * Command module for default ("run") command
+ */
+
+const path = require('path');
+const {loadConfig, findConfig} = require('./config');
```

Comment:
This aspect of `prettier` drives me nuts. When doing multi-imports via `require`, I _always_ want them listed vertically (like the L11-L21 version).